Winapp2.ini

```diff
@@ -142,6 +142,13 @@ FileKey12=%SystemDrive%\first_launch|*.*|REMOVESELF
 ExcludeKey1=FILE|%LocalAppData%\COMODO\Dragon\User Data\|First Run
 ExcludeKey2=FILE|%LocalAppData%\COMODO\Dragon\User Data\|Local State
 
+[Downloads Folder *]
+LangSecRef/Section=3025
+Warning=This will clear all the content of the default web browser downloads folder, including all old downloaded files.
+Default=False
+FileKey1=%UserProfile%\Downloads\|*.*|REMOVESELF
+FileKey2=%UserProfile%\Downloads\|*.*|RECURSE
+
```

Comment:
There is no need to use RECURSE and REMOVESELF together. You have to choose the correct one. For a system folder like "Downloads" it's not a good idea to use REMOVESELF, because this can break the folders function, especially on non-English systems.

Please also check the official documentation for the different file mask options: [CCleaner Documentation](https://www.ccleaner.com/docs/ccleaner/advanced-usage/ccleaner-ini-files/how-to-add-your-own-program-for-ccleaner-to-clean).